week-2/implement/bmi.js

```diff
@@ -13,3 +13,12 @@
 // Given someone's weight in kg and height in metres
 // When we call this function with the weight and height
 // Then it returns their Body Mass Index to 1 decimal place
+
+function calculateBMI(weight, height) {
+  const bmi = weight / (height * height);
+  return parseFloat(bmi.toFixed(1));
+  // Round to 1 decimal place (parseFloat(bmi.toFixed(1)) takes the calculated BMI, rounds it to one decimal place as a string, and then converts it back to a floating-point number, ensuring that the BMI returned by the function has only one decimal place. This helps keep the BMI value precise to the specified decimal point.)
```

Comment:
Could you explain why you used parseFloat here? I have not used it and that is why I am wondering what parseFloat would do.

Reply:
Thank you for your review @HadikaMalik 
parseFloat() is a  used to parse (or convert) a string into a floating-point number. 
it is used to ensure that the result of toFixed(1) (which rounds the BMI to one decimal place and returns it as a string) is converted back into a floating-point number.
For instance, if toFixed(1) returns "24.5" (as a string), using parseFloat("24.5") will convert it to the number 24.5.